app/src/main/java/com/example/development/sakaiclientandroid/utils/custom/TreeViewItemClickListener.java

```diff
@@ -23,16 +23,12 @@ public void onClick(TreeNode node, Object value) {
         if(node.getLevel() > 1)
             return;
 
+
         // Get all parent nodes
         for( TreeNode parent : root.getChildren()) {
             // If the parent was previously expanded and is not a parent
             // of the currently expanded node, then collapse it
-            if(parent.isExpanded() && parent == lastParent && parent != node) {
-                // Collapse its children as well
-                // for (TreeNode child : parent.getChildren()) {
-                //    if(child.isExpanded())
-                //        treeView.collapseNode(child);
-                // }
+            if(parent.isExpanded() && parent != node) {
```

Comment:
@akench since we're no longer using `lastParent` can you remove all references of it in this class?